recipe/meta.yaml

```diff
@@ -1,43 +1,58 @@
-{% set name = "jupyter-server-proxy" %}
-{% set version = "3.1.0" %}
+{% set version = "3.2.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: jupyter-server-proxy
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 30d57976090980f4380cb0a80e8e829dfd74d0a19edb64329239def9feefcde4
+  url: https://pypi.io/packages/source/j/jupyter-server-proxy/jupyter-server-proxy-{{ version }}.tar.gz
+  sha256: 816262f5711432dd79cbbd81e427cc2b592f522b0fae54368ed6e9d943384327
 
 build:
   number: 0
   noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  script: {{ PYTHON }} -m pip install --no-deps -vv --install-option="--skip-npm" .
 
 requirements:
   host:
-    - python >=3
+    - jupyter-packaging >=0.7,<0.8
+    - jupyterlab >=3.0,<4
     - pip
+    - python >=3.6
     - setuptools
-    - jupyter-packaging
   run:
-    - python >=3
-    - jupyter_server >=1
-    - simpervisor >=0.2
     - aiohttp
+    - jupyter_server >=1
+    - python >=3.6
+    - simpervisor >=0.4
```

Comment:
Should we hot-fix the old packages to have this constraint as well given it was [present in upstream since 3.0.0]( https://github.com/jupyterhub/jupyter-server-proxy/blob/4949a7eaf90c20a1c629e7f9420e8772464dc0d8/setup.py#L90 )?

Reply:
is this desired before this one is merged?

Reply:
It can be done after